### ISSUE TITLE
Error on deck.deal

### ISSUE THREAD

**Issue by daniwaxman** [troygoode/node-shuffle#5]
I am running a simple example:

var Shuffle = require("shuffle");
var deck = Shuffle.shuffle();
var card = deck.draw();

var player1, player2;
deck.deal(5, [player1, player2]);

console.log(player1, player2);
console.log(deck.length);

This results in an error on line 16 of deck.js  TypeError: Cannot read property 'push' of undefined

**Comment by AlexCPU** [comment]
You haven't defined player1 and player2, they need to be arrays.  This should work:
```
var Shuffle = require("shuffle");
var deck = Shuffle.shuffle();
var card = deck.draw();

var player1 = new Array;
var player2 = new Array;
deck.deal(5, [player1, player2]);

console.log(player1, player2);
console.log(deck.length);
```

**Comment by troygoode** [comment]
@daniwaxman as @AlexCPU pointed out, you'd need to pass in an initialized array for each hand. I've updated the README to clarify this (it incorrectly showed passing in uninitialized arrays in the example). I've also updated the `deal` function to throw a helpful error message if anyone forgets.

Updates available in `0.2.4`.
